pandas/tseries/tests/test_resample.py

```diff
@@ -361,6 +361,29 @@ def test_fillna(self):
         with self.assertRaises(ValueError):
             r.fillna(0)
 
+    def test_fill_value(self):
```

Comment:
Upsampling on a PeriodIndex with kind='period' doesn't work through reindex.   I think it would involve adding fill_value to Index.get_indexer (line 901 of tseries/resample.py) Before I start going down that path, I just want to confirm that's its the right thing to do.